roles/hammerdb/templates/db_mssql_workload_vm.sh.j2

```diff
@@ -54,8 +54,9 @@ export HOME=/root;
     or workload_args.client_vm.hostpath is sameas true %}
 sudo systemctl stop mssql-server;
 sudo ACCEPT_EULA='Y' MSSQL_PID='Enterprise' MSSQL_SA_PASSWORD='s3curePasswordString' /opt/mssql/bin/mssql-conf setup;
-sudo firewall-cmd --zone=public --add-port=1433/tcp --permanent;
-sudo firewall-cmd --reload;
+sudo firewall-offline-cmd --add-port=1433/tcp;
```

Comment:
Another option would be to simply move the firewall-cmd below the systemctl start firewalld.

Reply:
@RobertKrawitz, Tried it, **didnt work**. 
For more details [see](https://forums.centos.org/viewtopic.php?f=47&t=52162&p=220915)

Reply:
Fair enough, please add a comment about that so nobody misunderstands later.

Reply:
I preferred not to add comment inside this Jinja file (too sensitive)

Reply:
That's simply shell script code.  You can add a comment there.

Reply:
Add comment and test it locally